src/hotspot/share/gc/shared/collectedHeap.hpp

```diff
@@ -239,6 +243,11 @@ class CollectedHeap : public CHeapObj<mtInternal> {
   // Returns unused capacity.
   virtual size_t unused() const;
 
+  // Historic gc information
+  size_t free_at_last_gc()    { return _capacity_at_last_gc - _used_at_last_gc; }
+  size_t used_at_last_gc()    { return _used_at_last_gc; }
```

Comment:
There are some unnecessary spaces between () and {. Could you also make these functions const?